x-pack/legacy/plugins/uptime/server/graphql/monitors/schema.gql.ts

```diff
@@ -10,7 +10,7 @@ export const monitorsSchema = gql`
   "The data used to enrich the filter bar."
   type FilterBar {
     "A series of monitor IDs in the heartbeat indices."
-    ids: [MonitorKey!]
```

Comment:
Does this mean we can also delete the `MonitorKey` type now? Or is it present elsewhere?

Reply:
It is present elsewhere, I feel like that should be a subsequent PR since this one is targeting 7.3. I only removed the parts where it is part of this bug.